src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacCoordinateValidator.java

```diff
@@ -192,6 +199,9 @@ List<CoordinateItem> orderCoordinates(final List<CoordinateItem> items) {
             for (int i = 0; i < numDimensions; i++) {
                 String valueA = a.getCoordinates().get(i);
                 String valueB = b.getCoordinates().get(i);
+                if (valueA.isEmpty() || valueB.isEmpty()) {
+                    return 0;
```

Comment:
Agreed - sorting the empty strings to the start of the list affects the feedback that is given in some cases (since we'll always check empty values before anything else), but that's probably an improvement.